src/main/java/io/ebeaninternal/server/persist/BatchedPstmtHolder.java

```diff
@@ -53,7 +52,7 @@ public PreparedStatement getStmt(String stmtKey, BatchPostExecute postExecute) {
     if (bsSize > maxSize) {
       maxSize = bsSize;
     }
-    return bs.getStatement();
+    return bs;
```

Comment:
here the BatchedPstmt is returned. That will be passed to the DataBind, so that it can register the streams on the BatchedPstmt